pkgs/development/python-modules/django-hcaptcha/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, django, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "django-hCaptcha";
```

```suggestion
  pname = "django-hcaptcha";
```
we normalise the attr and pname